infra/resources.bicep

```
@@ -45,73 +42,53 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2019-06-01' = {
     minimumTlsVersion: 'TLS1_2'
     allowBlobPublicAccess: false
```

```suggestion
    allowBlobPublicAccess: false
    allowSharedKeyAccess: false
```
Suggest disabling allowSharedKeyAccess to force the storage account to only use identity and never keys.